play-services-fido-api/src/main/java/com/google/android/gms/fido/fido2/api/common/FidoAppIdExtension.java

```diff
@@ -31,6 +31,10 @@ public String getAppId() {
         return appId;
     }
 
+    public FidoAppIdExtension(String appId) {
```

Comment:
The deserializer needs a no-parameter constructor (which can be private though). Please add an additional private no-parameter constructor.